server/settings_test.go

```diff
@@ -41,6 +42,57 @@ func TestSettings(t *testing.T) {
 
 			defer restoreSettingsDefaults(t)
 
+			t.Run("InvalidBothEnableAndDisableAlerting", func(t *testing.T) {
```

Comment:
todo add a test case for successful changes in IA settings with slack and email config